modules/Vtiger/models/ExportToSpreadsheet.php

```diff
@@ -153,7 +153,7 @@ public function putDataIntoSpreadsheet(Vtiger_Field_Model $fieldModel, $value, i
 				}
 				break;
 			default:
-				$displayValue = $this->getDisplayValue($fieldModel, $value, $id, []) ?: '';
+				$displayValue = $this->getDisplayValue($fieldModel, $value, $id, []);
```

Comment:
$displayValue = $this->getDisplayValue($fieldModel, $value, $id, []) ?? '';
or cast to string